src/tutorial/meshvis.md

```diff
@@ -55,47 +53,56 @@ Rc=L/10;
 
+The first line from the above snippet defines physical volume 1 to coincide with the geometry volume 50, which is the final volume obtained by the Boolean operation. The second line defines physical surface 1 to include geometric surfaces {1,6,8}. Finally, the last line specifies the file format. Note that MFEM can only read ASCII Gmsh format version 2.2.
 
 <img style="width:60%" src="../img/gmsh02.png"> 
 <img style="width:60%" src="../img/gmsh03.png">
 
-The generated mesh is shown in the figure above. Careful inspection reveals that the cylindrical surface is not represented well by the linear elements. We can improve the representation by refining the mesh. The Gmsh input file can be downloaded  <a href="../cross_heat.geo">here</a>  and the mesh <a href="../cross_heat.msh">here</a>. The users are strongly encouraged to play with the mesh and to generate finer discretizations for the simulations.  
+The generated mesh is shown in the figures above. Careful inspection reveals that the cylindrical surface is not represented well by the linear elements. We can improve the representation by refining the mesh. We encourage you to play with the mesh and to generate finer discretizations for the simulations.  
+
+You can download the Gmsh input file [here](../cross_heat.geo) and the mesh [here](../cross_heat.msh). For users without access to the Gmsh GUI, a mesh can be generated in a terminal with the following command:
 
-For users without access to the graphical interface of Gmsh, a mesh can be generated in a terminal by the following command: 
 ```diff
 gmsh -3 cross_heat.geo
 ```
 
-To run simulations with the generated mesh, go to the web browser, click on the "Explorer" window with the right bottom of the mouse and select Upload to upload the mesh file from your computer to the AWS machine. Once uploaded, the file is available to any program on the AWS machine. To run example 1 with the newly prepared mesh, copy the file to the examples directory and run the parallel version of example 1 with the following:
+To run simulations with the generated mesh, drag-and-drop the mesh file from your computer to the AWS browser window in the MFEM `examples` directory. To run Example 1 with the newly prepared mesh, be sure you are in the `examples` directory and then run the appropriate command:
+
 ```diff
-euler@26c14060d771:~/mfem/examples$ mpirun -np 24 ./ex1p -m mesh_file.msh
+mpirun -np 24 ./ex1p -m cross_heat.geo
+mpirun -np 24 ./ex1p -m cross_heat.msh
 ``` 
-The solution of the diffusion equation for the generated mesh is shown in the following two pictures. The figures are generated with ParaView, and the process of visualization is explained at the end of this tutorial session. 
+
+The solution of the diffusion equation for the generated mesh is shown in the following two pictures. The figures are generated with ParaView, and the process of visualization is explained at the end of this tutorial session.
```

Comment:
Do I need to install ParaView in order to see this viz? Or add the lines from the "Visualizing results in VisIt and ParaView" section?